src/promptflow/promptflow/_core/tools_manager.py

```diff
@@ -120,6 +150,7 @@ def collect_package_tools_and_connections(keys: Optional[List[str]] = None) -> d
                 module = importlib.import_module(m)  # Import the module to make sure it is valid
                 tool["package"] = entry_point.dist.metadata["Name"]
                 tool["package_version"] = entry_point.dist.version
+                set_default_input_index_for_tool(tool)
```

Comment:
method name change to try_to_set_input_index_for_tool ?

Reply:
Changed, thanks.